README.md

```diff
@@ -44,6 +44,7 @@ The benchmark can be configured through the following environment variables:
 |**Name**|**Description**|**Default value**|
 |--------|---------------|:---------------:|
 |GRPC_BENCHMARK_DURATION|Duration of the benchmark.|30s|
+|GRPC_BENCHMARK_WARMUP|Duration of the warmup. Stats won't be collected.|30s|
```

Comment:
Do we perhaps want to override the warmup in `CI` to smaller value?

Reply:
If we are going to have specific values in the CI workflow, how about we change the defaults in the shell script to values that we believe people should be using on a first shot at these benchmarks?

DURATION=300s
WARMUP=30s

For CI, we use:

DURATION=30s
WARMUP=15s

Wdyt?

Reply:
@brunoborges This sounds good, I'd even decrease it further for the CI, e.g. 5s and 20/25s. The CI is only there to check if the benchmarks / services work at all, the Github Actions results are not reliable anyway.

Reply:
Cool. I will add logic for the WARM UP, that if it is zero, it won't run.

I'll change the defaults to what we believe is best too, while adding low values for the CI workflow.